safe/utilities/test/test_clipper.py

```diff
@@ -666,6 +668,40 @@ def test_adjust_clip_extent(self):
             expected_adjusted_extent, adjusted_extent)
         self.assertEqual(adjusted_extent, expected_adjusted_extent, message)
 
+    def test_clip_vector_with_unicode(self):
+        """Test clipping vector layer with unicode attribute in feature.
+
+        This issue is described at Github #2262 and #2233
+        """
```

Comment:
I added a new test here.

This is to test that the clipped layer should contains possible values from original layer, to make sure it is consitent. Originally this test fails when the attribute compared contains unicode char, which in turn make Impact Layer Merge fails.

But the test succeeded if we write the attribute using QgsVectorFileWriter by specifying None in the encoding.
Should we just put None for the encoding?